lib/bitcoin/fee/SlidingWindowQuantileCalculator.ts

```diff
@@ -177,7 +183,9 @@ export default class SlidingWindowQuantileCalculator {
    * Get the quantile as of a specific groupId.
    */
   public getQuantile (groupId: number): number | undefined {
-    return this.historicalQuantiles.get(groupId);
+    const quantile = this.historicalQuantiles.get(groupId);
+
+    return quantile ? Math.ceil(quantile) : undefined;
```

Comment:
Sanity: quantile is expressed in satoshis right?

Reply:
The `.round()` call was removed from the `ValueApproximator` but we want to keep the fraction values as now we have a window (.1%) of deviation allowed. Also the `ValueApproximator` does not look at this as satoshis but just some value. If anything either the `getQuantile` output (like here) or the caller of this function should round the values as the values are expressed as satoshis only at that level.

Reply:
Agree with you that should not put it in `ValueApproximator`, but then I don't like that there are two definitions or quantiles. One "fake ones" that `historicalQuantiles` stores, and "real ones" that `getQuantile` returns. Maybe we should make sure they both mean the same thing by computing the ceiling at write time in `calculateCurrentQuantile()`?

Reply:
The 'current' quantile is only allowed to deviate a limited from the previous vlaues. We use the `historicalQuantiles` to get the previous value so saving the rounded values in there will affect that comparison. Now off course, we can rearchitect to save the previous value separately in my mind, it is better that internally we use the 'non-rounded' quantiles and only expose the rounded value to the public callers. To me it is more confusing to save the rounded values in `historicalQuantiles` and then non-rounded ones at other places such as `valueApproximator` etc.